en/SUMMARY.md

```diff
@@ -88,6 +88,7 @@
     * [Landing \(Fixed Wing\)](flying/fixed_wing_landing.md)
     * [Takeoff \(Fixed Wing\)](flying/fixed_wing_takeoff.md)
   * [Missions](flying/missions.md)
+    * [Package Delivery Mission Planning](flying/package_delivery_mission.md)
```

Comment:
Removing "Planning" - it is inferred - and in sidebar we benefit from shorter titles.
```suggestion
    * [Package Delivery Mission](flying/package_delivery_mission.md)
```